### PR TITLE
Add initial NuGet pipeline for Windows x64 build

### PR DESCRIPTION
**Pull request by sayanshaw24**

### CODE REVIEW

File: .pipelines/nuget.yml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        )
+
+        @echo %vsdevcmd% will be used as the VC compiler
+        @echo ##vso[task.setvariable variable=vsdevcmd]%vsdevcmd%
+      displayName: 'locate vsdevcmd via vswhere'
+
+    # Build Extensions for Windows x64 - we will add more builds to the nuget pipeline in the future
+    - script: |
+        call $(vsdevcmd)
+        call .\build.bat -DOCOS_ENABLE_CTEST=ON -DONNXRUNTIME_PKG_DIR=.\onnxruntime-win-x64-$(ort.version) -DOCOS_ENABLE_BLINGFIRE=OFF
```
Comment (wenbingl) [inline .pipelines/nuget.yml:54]: maybe not this PR, but can you add abuild.ps1 or build.bat in tools folder to create 4 build architecture arm, arm64, x64 and x86. so we can create the full Windows build package here.

File: .pipelines/nuget.yml [hunk 2]
```diff
@@ -0,0 +1,100 @@
+stages:
+- stage: NuGet_Packaging_CPU
+  dependsOn:
+  condition: succeeded()
```
Comment (wenbingl) [inline .pipelines/nuget.yml:4]: can be removed

File: .pipelines/templates/component-governance-component-detection-steps.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+- ${{ if eq(variables['System.TeamProject'], 'Lotus') }}: 
+  - task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
+    displayName: 'Component Detection'
+    condition:
+      or(or(and(eq('${{parameters.condition}}', 'ci_only'), and(succeeded(), in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'Scheduled'))),
+          and(eq('${{parameters.condition}}', 'always'), always())),
+          and(eq('${{parameters.condition}}', 'succeeded'), succeeded()))
+    inputs:
+      # ignore dmlc-core tracker for its CI, which is not used in onnxruntime build
+      # ignore unit tests in emscripten. emscripten unit tests are not used in onnxruntime build
```
Comment (wenbingl) [inline .pipelines/templates/component-governance-component-detection-steps.yml:17]: only apply to onnxruntime

**Review by wenbingl — DISMISSED** [review]

File: .pipelines/nuget.yml (hunk 1 above)
Comment (sayanshaw24) [reply .pipelines/nuget.yml:54]: yeah sounds good, will create a different PR for other architectures

File: .pipelines/nuget.yml (hunk 2 above)
Comment (sayanshaw24) [reply .pipelines/nuget.yml:4]: good catch, missed this, thanks!

File: nuget/NativeNuget.nuspec [hunk 4]
```diff
@@ -10,10 +10,11 @@
 	<!-- TODO: Update release notes prior to 0.8.0 release with new features here -->
 	<releaseNotes>
 		General
-		1. New custom operators: Whisper, YOLO, RobertaTokenizer, ClipTokenizer, EncodeImage, DecodeImage
-		2. ORT custom operator C++ stub generation tool
-		3. Operator implementation and documentation improved.
-		4. Python (3.7 - 3.10) and ORT (1.10 above) compatible.
+		1. New custom operators: Whisper, Bounding Box Drawing, RobertaTokenizer, ClipTokenizer, EncodeImage, DecodeImage
```
Comment (skottmckay) [inline nuget/NativeNuget.nuspec:13]:
```suggestion
		1. New custom operators: Whisper, DrawBoundingBoxes, RobertaTokenizer, ClipTokenizer, EncodeImage, DecodeImage
```

FWIW EncodeImage and DecodeImage were in the previous release

Comment (sayanshaw24) [reply nuget/NativeNuget.nuspec:13]: for now, i'm just adding all new features on top of 0.7.0 release notes prior to official 0.8.0 release.

**Review by skottmckay — APPROVED** [review]

**Pull request merged**